modules/sdk-coin-apt/src/lib/transactionBuilderFactory.ts

```diff
@@ -19,7 +19,7 @@ export class TransactionBuilderFactory extends BaseTransactionBuilderFactory {
   /** @inheritdoc */
   from(signedRawTxn: string): TransactionBuilder {
     try {
-      const signedTxn = this.parseTransaction(signedRawTxn);
+      const signedTxn = utils.parseTransaction(signedRawTxn);
       const txnType = this.getTransactionTypeFromSignedTxn(signedTxn);
```

Comment:
can we extract a util function for these two statements and reuse that function here and in `getTransactionFromRawTx()` method?